package.json

```
@@ -9,10 +9,10 @@
     "lint": "npm run lint:eslint && npm run lint:prettier",
     "lint:fix": "npm run lint:eslint -- --fix && npm run lint:prettier -- --write",
     "build": "npm run build:sass && npm run build:js && npm run copy-assets",
-    "build:js": "npm run build:js:application && npm run build:js:cookies && npm run build:js:all",
+    "build:js": "yarn build:js:application && yarn build:js:analytics && yarn build:js:all",
     "build:js:all": "mkdir -p dist/public/javascripts; uglifyjs node_modules/govuk-frontend/govuk/all.js node_modules/hmpo-components/all.js --beautify -o dist/public/javascripts/all.js",
+    "build:js:analytics": "mkdir -p dist/public/javascripts; uglifyjs node_modules/@govuk-one-login/di-ipv-cri-common-express/src/assets/javascript/analytics/**/*.js node_modules/@govuk-one-login/di-ipv-cri-common-express/src/assets/javascript/analytics/ua/legacy/cookies.js node_modules/@govuk-one-login/di-ipv-cri-common-express/src/assets/javascript/analytics/*.js --beautify -o dist/public/javascripts/analytics.js",
```

```suggestion
    "build:js:analytics": "mkdir -p dist/public/javascripts; shopt -s globstar; uglifyjs node_modules/@govuk-one-login/di-ipv-cri-common-express/src/assets/javascript/analytics/**/*.js --beautify -o dist/public/javascripts/analytics.js",
```
This command is unwieldy and not easy to maintain if the filepaths are hardcoded since they can change. Instead we can use the globstar option to recursively get all the js files (including in the root of the tree, i.e. `.../analytics/*.js`).

I am not sure if we want to also include the `init.js` files - if not we can exlude them with `.../analytics/**/!(init).js` - this will get all the js files except the ones named `**/init.js`.

---

As discussed in standup, although this is a valid concern, it's out of scope for this ticket, and applies to all three CRI teams, so should be dealt with as part of tech debt.